docs/pages/management/diagnostics/profiles.mdx

```diff
@@ -7,20 +7,51 @@ Teleport leverages Go's diagnostic capabilities to collect and export
 profiling data. Profiles can help identify the cause of spikes in CPU,
 the source of memory leaks, or the reason for a deadlock.
 
-## Enable profiling
+## Using `teleport debug profile`
+
+(!docs/pages/includes/diagnostics/debug-service-intro.mdx!)
+
+`teleport debug profile` collects a list of pprof profiles. It outputs a
+compressed tarball (`.tar.gz`) to STDOUT. You decompress it using `tar` or
+direct the result to a file.
+
+By default, it collects `goroutine`, `heap` and `profile` profiles.
+
+```bash
+# Collect default profiles and save to a file.
+$ teleport debug profile > pprof.tar.gz
+$ tar xvf pprof.tar.gz
+
+# Collect default profiles and decompress it.
+$ teleport debug profile | tar xzv -C ./
+
+# Collect "trace" and "mutex" profiles and save to a file.
+$ teleport debug profile trace,mutex > pprof.tar.gz
+```
```

Comment:
Could we add an example (or reuse one of the examples above) with `seconds`?

nit: I also really like examples below where we show the `go tool pprof` command to visualize the result and i think it would be great if we can add them here too.

Reply:
We should link the CLI reference for `teleport debug` somewhere in this page.

Reply:
I've added examples below this bash block.